.template/addons/github/.github/wiki/_Sidebar.md

```diff
@@ -2,6 +2,7 @@
 
 - [[Home]]
 - [[Getting Started]]
+- [[Contribution_Guide]]
```

Comment:
Should we use it without the `underscore`, it's more readable indeed!!!

```suggestion
- [[Contribution Guide]]
```

Reply:
Please also update the tests. Thanks!!!

Reply:
Thanks. Fixed in f6a068eaa9bd57a1bdccf34b313b9b56fd7d7290